source/releasenotes/2024-05-02-Synthetics-Check-Limits.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "New Relic Synthetics: New monthly Check limits Starting May 1st, 2024"
+published_date: "2024-05-02"
+categories: [action-required]
+---
+**New Relic Synthetics: New monthly Check limits**
```

Comment:
This line isn't necessary since the title will be published

Reply:
Updated.